networktest.go

```diff
@@ -187,6 +193,26 @@ func (l *lightningNode) Start(lndError chan error) error {
 	var errb bytes.Buffer
 	l.cmd.Stderr = &errb
 
+	// If the logoutput flag is passed, redirect output from the nodes to
+	// log files.
+	if *logOutput {
```

Comment:
In light of the comments above, this should be something like:
```go
if *logOutput != -1 && l.nodeID == *logOutput{
```

Perhaps we should also change the name of the flag variable in order to better reflect its current semantics?